document_page_approval/models/document_page.py

```diff
@@ -48,6 +46,7 @@ class DocumentPage(models.Model):
         "Approval required",
         help="If true, changes of this page require approval",
         compute="_compute_is_approval_required",
+        recursive=True,
```

Comment:
Why do you need this?

Reply:
Without this change, when executing the tests, the following log is shown:

`WARNING devel py.warnings: /opt/odoo/custom/src/odoo/odoo/fields.py:722: UserWarning: Field document.page.approver_group_ids should be declared with recursive=True`

Reply:
Uhm, OK, but I don't know why